CONTRIBUTING.md

```diff
@@ -59,9 +59,11 @@ We're happy to help!:raised_hands:
 
 1. Fork the [supertokens-core](https://github.com/supertokens/supertokens-core) repository (**Skip this step if you are
    NOT modifying supertokens-core**)
-2. `git clone https://github.com/supertokens/supertokens-root.git`
-3. `cd supertokens-root`
-4. Open the `modules.txt` file in an editor (**Skip this step if you are NOT modifying supertokens-core**):
+2. Fork the [plugin-interface](https://github.com/supertokens/supertokens-plugin-interface) repository (**Skip this step if you are

```

Comment:
I don't think that this step is necessary to do for contribution. Therefore it wasn't mentioned in the first place.